conf/layer.conf

```diff
@@ -8,7 +8,7 @@ BBFILES += "${LAYERDIR}/recipes-*/*/*.bb \
 BBFILE_COLLECTIONS += "celebrity-face-match"
 BBFILE_PATTERN_celebrity-face-match = "^${LAYERDIR}/"
 BBFILE_PRIORITY_celebrity-face-match = "24"
-LAYERSERIES_COMPAT_celebrity-face-match = "hardknott"
+LAYERSERIES_COMPAT_celebrity-face-match = "kirkstone"
 LAYERDEPENDS_celebrity-face-match = "phytec ampliphy meta-ml"
 
-DISTRO_VERSION_append = "-CelebrityFaceMatch-v0.6"
+DISTRO_VERSION:append = "-CelebrityFaceMatch-v0.7"
```

Comment:
Please put this in a separate update version patch which concludes all your update patches.

Reply:
Well but it is an append to the distro version and we switch from hardknott to kirkstone. So it is already something like 0.7.y or 0.7-dev. Agree, that the final 0.7 should conclude ...

Reply:
What about `-CelebrityFaceMatch-kirkstone-dev`?

Reply:
Not sure, duplicate encoding as PD23.1.0 already contains this somehow. And string gets longer and longer...

Reply:
Ok, then `0.7-dev`.